code/tests/files/torch/torch.q

```diff
@@ -1,13 +1,16 @@
+// code/tests/files/torch/torch.q - PyTorch test files
+// Copyright (c) 2021 Kx Systems Inc
+
```

Comment:
Extra description can be added `Contains the functionality to apply vanilla pytorch models within the automl framework`

Reply:
added description